### ISSUE TITLE
"Set up in Desktop" from the Web creates an "empty repo" error.

### ISSUE THREAD

**Issue by justalemon** [desktop/desktop#2897]
### Description

If you create a repo from scratch on the web (no licence, gitignore or readme) and then you try to "Set it up" on Github Desktop you will get:

```
warning: You appear to have cloned an empty repository.
Could not pull
```

and you will have to add the local repository manually (File > Add local repository).

### Version

**GitHub Desktop version:** 1.0.1
**OS version:** Microsoft Windows 10.0.15063

### Steps to Reproduce

1. Create a empty repository on the Web ([example](https://github.com/Lemon-CL/ScriptsGTA))
1. Press "Set up in Desktop" or search for it on the app
1. Clone the repo

**Expected behavior:** Repo added on the app, because is cloned without problems.

**Actual behavior:**
![image](https://user-images.githubusercontent.com/11861253/30953796-de2fa926-a403-11e7-88b9-efebf0cf6943.png)

**Reproduces how often:** 100%

### Logs

[2017-09-28.desktop.production.log](https://github.com/desktop/desktop/files/1339823/2017-09-28.desktop.production.log)
[20170928T041800.6498129.log](https://github.com/desktop/desktop/files/1339844/20170928T041800.6498129.log)

**Comment by shiftkey** [comment]
This was a bug in Git LFS that is fixed in 2.3.1 - https://github.com/git-lfs/git-lfs/issues/2592 - we need to update our Git tooling to make this go away.

**Comment by justalemon** [comment]
Nice, thanks for that.
